### PR TITLE
Feauture structure define

### PR DESCRIPTION
**Pull request by LauEroles**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	projectLead
+	manager
+	vp
+	director
+	cto
+
+ )
+*/
+
+//Defined a type of ths date structure person
```
Comment (LDiegoM) [inline main.go:33]: You have a typo here
```suggestion
//Defined a type of this date structure person
```

File: main.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+	country         string
+	linkedinProfile string
+	email           string
+	speciality      string
+	//Type		SeniorityType
+	techs        string
+	observations string
+}
+
+//Defined a type of ths date structure contact
```
Comment (LDiegoM) [inline main.go:46]: Another typo here :)
```suggestion
//Defined a type of this date structure contact
```

File: main.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+type contact struct {
+	name     string
+	contacts []person
+}
+
+//Inicializar contact( a slice of  person)
```
Comment (LDiegoM) [inline main.go:53]: Spanglish?

File: main.go [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+	contactList := contact{
+		name:     "Agenda",
+		contacts: []person{person1, person2, person3},
+	}
+	return contactList
+}
+
+func showContacts(oneContact contact) {
+	fmt.Println(" The agenda have the following contacts: ")
+	/*for i, ingrediente := range unaReceta.Ingredientes {
```
Comment (LDiegoM) [inline main.go:95]: Think you can remove this unused code

File: main.go [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+		fmt.Println("LinkedIn, " + agenda.linkedinProfile)
+		fmt.Println("email, " + agenda.email)
+		fmt.Println("speciality, " + agenda.speciality)
+		fmt.Println(" tech, " + agenda.techs)
+		fmt.Println("Observations, " + agenda.observations)
+	}
+
+}
+
+/*func showPerson(aPerson person) {
```
Comment (LDiegoM) [inline main.go:115]: You can remove this unused code also

File: main.go (hunk 1 above)
Comment (LauEroles) [reply main.go:33]: tks! changed!

File: main.go (hunk 2 above)
Comment (LauEroles) [reply main.go:46]: thanks again! changed :) !

File: main.go (hunk 3 above)
Comment (LauEroles) [reply main.go:53]: OMG, I think so.. changing

File: main.go (hunk 4 above)
Comment (LauEroles) [reply main.go:95]: done! 👍

File: main.go (hunk 5 above)
Comment (LauEroles) [reply main.go:115]: 😄

**Pull request merged**